huma_signals/dotenv/example.env

```diff
@@ -1,16 +0,0 @@
-CHAIN=GOERLI
-
```

Comment:
Can we bring this back please? Seems to be accidentally deleted.

Reply:
Yes absolutely